src/progpy/data_models/lstm_model.py

```diff
@@ -476,7 +476,7 @@ def from_data(cls, inputs, outputs, event_states=None, t_met=None, **kwargs):
             raise ValueError(f"layers must be greater than 0, got {params['layers']}")
         if np.isscalar(params['units']):
             params['units'] = [params['units'] for _ in range(params['layers'])]
-        if not isinstance(params['units'], (list, np.ndarray)):
+        if not isinstance(params['units'], (list, np.ndarray, tuple)):
             raise TypeError(f"units must be a list of integers, not {type(params['units'])}")
```

Comment:
Should we change the exceptions to mention that 'units must be a sequence of integers' instead of 'a list'?